client/src/js/util.js

```diff
@@ -0,0 +1,2 @@
+export const $ = document.querySelector.bind(document);
+export const $$ = document.querySelectorAll.bind(document);
```

Comment:
arrow function 사용하면 bind 해주지 않아도 사용 가능하겠네요!

```suggestion
export const $$ = (selector) => document.querySelectorAll(selector);
```